mitreattack/navlayers/core/README.md

```diff
@@ -34,8 +34,8 @@ layer_example.layer.versions = dict(layer="4.2", attack="9.1", navigator="4.2")
 layer_example.layer.description = "This is a demonstration of how to set up a layer piece by piece"
 
 # configure the "filters" object
-layer_example.layer.filters = dict(platforms=['example'])  # platforms can be provided during initialization
-layer_example.layer.filters.platforms = ['a', 'b', 'c']  # or separately
+layer_example.layer.filters = dict(platforms=['MacOS'])  # platforms can be provided during initialization
```

Comment:
`macOS` should not start with a capital `M`: https://github.com/mitre-attack/attack-navigator/blob/develop/layers/LAYERFORMATv4_2.md#filter-object-properties

Windows (below) is fine, it's just `macOS` and iOS that don't start with a capital.

Reply:
Yep, that's on me. Thanks for catching that.